core/storage/story/gae_models_test.py

```diff
@@ -106,6 +136,35 @@ def test_get_by_url_fragment(self) -> None:
 class StoryCommitLogEntryModelUnitTest(test_utils.GenericTestBase):
     """Test the StoryCommitLogEntryModel class."""
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = {
+            'story_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'commit_cmds': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'commit_message':
+            base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Indentation is wrong, here and elsewhere. Use parens and indent by 4. See [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) for examples of how this should look (you can add a new paren around base_models.EXPORT_POLICY.NOT_APPLICABLE).